lib/tests/streamlit/runtime/state/widgets_test.py

```diff
@@ -587,6 +593,28 @@ def test_widget_id_computation_data_editor(self):
             st.data_editor(data=[], disabled=True)
 
 
+class RegisterWidgetsTest(DeltaGeneratorTestCase):
+    @parameterized.expand(WIDGET_ELEMENTS)
+    @unittest.skipIf(
+        sys.version_info < (3, 9), reason="the type check requires python3.9 or higher"
+    )
+    def test_register_widget_called_with_valid_value_type(
+        self, _element_name: str, widget_func: ELEMENT_PRODUCER
+    ):
+        with patch(
+            "streamlit.runtime.state.widgets.register_widget_from_metadata",
+            wraps=register_widget_from_metadata,
+        ) as patched_register_widget_from_metadata:
+            widget_func()
+        assert patched_register_widget_from_metadata.call_count == 1
+
+        # ValueFieldName already inclused ArrayValueFieldName, but the check fails
```

Comment:
inclused -> includes

Reply:
Oh that was a leftover comment. I removed it completely